admin_app/src/app/content/SearchSidebar.tsx

```diff
@@ -0,0 +1,388 @@
+import React, { useState } from "react";
```

Comment:
We have been putting components under /component. So suggest moving this to `content/components`